### PR TITLE
disfetch: init at 1.12

### PR DESCRIPTION
**Pull request by q60**
###### Motivation for this change
Disfetch is another *nix distro fetching program, but less complex. It is based on several principles
+ less code, more efficiency, more readability
+ simplicity
+ logos must be only 8 rows
+ using only coreutils, other built-ins, bash and nothing third-party
+ showing only main and needed system information

https://github.com/llathasa-veleth/disfetch

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -9275,6 +9275,12 @@
     githubId = 2212422;
     name = "uwap";
   };
+  v = {
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:9278]: Please put this into a separate commit with the message ``maintainers: add veleth`` and maybe change your name to veleth.

File: pkgs/tools/misc/disfetch/default.nix [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{ stdenv
+, lib
+, fetchFromGitHub }: stdenv.mkDerivation rec {
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/disfetch/default.nix:3]:
```suggestion
, fetchFromGitHub }:

stdenv.mkDerivation rec {
```

**Review by SuperSandro2000 — COMMENTED** [review]
Please squash the commits together.

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 107788` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>disfetch</li>
  </ul>
</details>

**Comment by llathasa-veleth** [comment]
Hi, should I close this pull request then make a new one with right steps? Sorry for incompetence.

**Comment by SuperSandro2000** [comment]
> Hi, should I close this pull request then make a new one with right steps?

No, you should do the changes and force push them.
